StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Link/LinkController.swift

```diff
@@ -391,13 +453,54 @@ import UIKit
     /// - Parameter email: The email address to look up.
     /// - Returns: Returns `true` if the email is associated with an existing Link consumer, or `false` otherwise.
     func lookupConsumer(with email: String) async throws -> Bool {
-        return try await withCheckedThrowingContinuation { continuation in
+        try await withCheckedThrowingContinuation { continuation in
             lookupConsumer(with: email) { result in
                 switch result {
                 case .success(let isExistingLinkConsumer):
                     continuation.resume(returning: isExistingLinkConsumer)
-                case .failure(let failure):
-                    continuation.resume(throwing: failure)
+                case .failure(let error):
+                    continuation.resume(throwing: error)
+                }
+            }
+        }
+    }
+
+    /// Registers a new Link user with the provided details.
+    /// `lookupConsumer` must be called before this.
+    ///
+    /// - Parameter fullName: The full name of the user.
+    /// - Parameter phone: The phone number of the user. Expected to be in E.164 format.
+    /// - Parameter country: The country code of the user.
+    /// Throws if `lookupConsumer` was not called prior to this, or an API error occurs.
+    func registerNewLinkUser(
+        fullName: String?,
+        phone: String,
+        country: String
+    ) async throws {
+        try await withCheckedThrowingContinuation { continuation in
+            registerNewLinkUser(fullName: fullName, phone: phone, country: country) { result in
+                switch result {
+                case .success:
+                    continuation.resume(returning: ())
+                case .failure(let error):
+                    continuation.resume(throwing: error)
+                }
+            }
+        }
+    }
+
+    /// Presents the Link verification flow for an existing user.
+    ///
+    /// - Parameter viewController: The view controller from which to present the authentication flow.
+    /// - Returns: An `AuthenticationResult` indicating whether authentication was completed or canceled.
```

Comment:
Since this doesn’t take `email` as a parameter, should the `presentForVerification` api docs also mention the requirement of calling `lookupConsumer` like you have for `registerNewLinkUser`, like the following?
```suggestion
    /// Presents the Link verification flow for an existing user.
    /// `lookupConsumer` must be called before this.
    ///
    /// - Parameter viewController: The view controller from which to present the authentication flow.
    /// - Returns: An `AuthenticationResult` indicating whether authentication was completed or canceled.
    /// Throws if `lookupConsumer` was not called prior to this, or an API error occurs.
```